tests/conftest.py

```diff
@@ -257,16 +290,19 @@ def default_2000_blocks_compact(bt):
 def default_10000_blocks_compact(bt):
     from tests.util.blockchain import persistent_blocks
 
-    return persistent_blocks(
-        10000,
-        f"test_blocks_10000_compact_{saved_blocks_version}.db",
-        bt,
-        normalized_to_identity_cc_eos=True,
-        normalized_to_identity_icc_eos=True,
-        normalized_to_identity_cc_ip=True,
-        normalized_to_identity_cc_sp=True,
-        seed=b"1000_compact",
-    )
+    if bt.constants.SOFT_FORK3_HEIGHT == 0:
+        pytest.skip("Test cache not available yet")
+    else:
```

Comment:
I don't think you need to put this in an else-block. you can preserve the indentation level